Makefile

```diff
@@ -45,9 +45,9 @@ mem:
 	go tool pprof --alloc_space pprof/fastime-test.bin pprof/mem-fastime.out
 
 lint:
-	gometalinter --enable-all . | rg -v comment
+	golangci-lint run
```

Comment:
if you switch to golangci, maybe add config as well?

Reply:
added